applications/vvo/tests/test_vvo_app.py

```diff
@@ -0,0 +1,361 @@
+	        },
+	        "nd_m1047574":{
+	            "voltage_A":"6306.632406-4741.568924j V",
+	            "voltage_B":"-7214.626338-2987.055914j V",
+	            "voltage_C":"622.058711+7442.125124j V"
+	        },
+	        "rcon_FEEDER_REG":{
+	            "Control":"MANUAL",
+	            "PT_phase":"CBA",
+	            "band_center":126.5,
+	            "band_width":2.0,
+	            "connect_type":"WYE_WYE",
+	            "control_level":"INDIVIDUAL",
+	            "dwell_time":15.0,
+	            "lower_taps":16,
+	            "raise_taps":16,
+	            "regulation":0.10000000000000001
+	        },
+	        "rcon_VREG2":{
+	            "Control":"MANUAL",
+	            "PT_phase":"CBA",
+	            "band_center":125.0,
+	            "band_width":2.0,
+	            "connect_type":"WYE_WYE",
+	            "control_level":"INDIVIDUAL",
+	            "dwell_time":15.0,
+	            "lower_taps":16,
+	            "raise_taps":16,
+	            "regulation":0.10000000000000001
+	        },
+	        "rcon_VREG3":{
+	            "Control":"MANUAL",
+	            "PT_phase":"CBA",
+	            "band_center":125.0,
+	            "band_width":2.0,
+	            "connect_type":"WYE_WYE",
+	            "control_level":"INDIVIDUAL",
+	            "dwell_time":15.0,
+	            "lower_taps":16,
+	            "raise_taps":16,
+	            "regulation":0.10000000000000001
+	        },
+	        "rcon_VREG4":{
+	            "Control":"MANUAL",
+	            "PT_phase":"CBA",
+	            "band_center":125.0,
+	            "band_width":2.0,
+	            "connect_type":"WYE_WYE",
+	            "control_level":"INDIVIDUAL",
+	            "dwell_time":15.0,
+	            "lower_taps":16,
+	            "raise_taps":16,
+	            "regulation":0.10000000000000001
+	        },
+	        "reg_FEEDER_REG":{
+	            "configuration":"rcon_FEEDER_REG",
+	            "phases":"ABC",
+	            "tap_A":2,
+	            "tap_B":2,
+	            "tap_C":1,
+	            "to":"nd__hvmv_sub_lsb"
+	        },
+	        "reg_VREG2":{
+	            "configuration":"rcon_VREG2",
+	            "phases":"ABC",
+	            "tap_A":10,
+	            "tap_B":6,
+	            "tap_C":2,
+	            "to":"nd_190-8593"
+	        },
+	        "reg_VREG3":{
+	            "configuration":"rcon_VREG3",
+	            "phases":"ABC",
+	            "tap_A":16,
+	            "tap_B":10,
+	            "tap_C":1,
+	            "to":"nd_190-8581"
+	        },
+	        "reg_VREG4":{
+	            "configuration":"rcon_VREG4",
+	            "phases":"ABC",
+	            "tap_A":12,
+	            "tap_B":12,
+	            "tap_C":5,
+	            "to":"nd_190-7361"
+	        },
+	        "xf_hvmv_sub":{
+	            "power_in_A":"1739729.121744-774784.928874j VA",
+	            "power_in_B":"1659762.622236-785218.729252j VA",
+	            "power_in_C":"1709521.679116-849734.584017j VA"
+	        }
+	    }
+	}
+    vvo_output_dict = {
```

Comment:
I think that this should be 'vvo_expected_output_dict' not to be confused with output dict within the function...This is a nit pick however so tell me to stuff it if you don't agree.